src/components/faceVerification/standalone/screens/LoginSuccessScreen.js

```diff
@@ -25,7 +27,7 @@ const LoginSuccessScreen = ({ styles }) => {
 
     const url = rdu || cbu
     const urlType = rdu ? 'rdu' : 'cbu'
-    const response = addNonceAndSign({
+    const response = addNonceAndSign(goodWallet, {
```

Comment:
i have no idea where this comes from, it wasnt in the original screen, fvflow is without wallet and doesnt need to sign anything plus please RENAME BACK all name changes you did to my original names.
if you still need to name things use FVFlow and not Login